content/frameworks/react/code-splitting-suspense/index.md

```diff
@@ -0,0 +1,150 @@
+---
```

Comment:
So quick general comment-- this guide is excellent. I really like seeing how to code split in React. I also like how you cover alternatives and edge cases. There's a bit of language leaning that I could suggest to clean things up, but I think I'll wait to do that in the next round of reviews, as it's fairly close as is.